Anna Gamzinova/HW6c_gamzinova.py

```diff
@@ -0,0 +1,12 @@
+# Counting how many times each letter appears in this file
+
+import re
+
+with open("text.txt", "r") as file:
+    text = file.read()
+# all the special symbols and digits removed
+text_updated = re.sub(r'\W|\d', r'', text)
+
+for letter in sorted(set(text_updated)):
+    print(f'{letter}: {text_updated.count(letter)}')
```

Comment:
interesting solution. Good.
Alternative way:

import re
with open("text.txt", 'r') as file:
text = file.read()
regex = re.compile('[^A-z]')
result = list(regex.sub('', text).lower())
resulted_dict = {}.fromkeys(result)
for key in resulted_dict.keys():
resulted_dict[key] = result.count(key)
print(resulted_dict)